keyboards/handwired/atreus50/keymaps/ajp10304/readme.md

```diff
@@ -1,11 +1,14 @@
 # AJP10304 Custom Atreus50 Layout
-# Also available for the Planck and JJ40
+# Also available for the Planck, Shark and JJ40
 
 **Note:** In the tables below where there are two characters on a key,
 the second is the output when shift is applied.
 
 **Note:** The below tables assume a UK layout.
 
+#### Flashing
+teensy_loader_cli -w -mmcu=atmega32u4 handwired_atreus50_ajp10304.hex
```

Comment:
you should be able to just use the `:flash` target when compiling.  This will compile and flash the board, all in one go.